frametrim/tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+
+MACRO(TRACE_TEST name frame orig_img)
+  
+  ADD_TEST(NAME ${name}-${frame} COMMAND
+    /bin/bash ${CMAKE_CURRENT_SOURCE_DIR}/compare-trace.sh
+    ${CMAKE_BINARY_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${name}.test_trace
+    ${frame}
+    ${orig_img})
+ENDMACRO(TRACE_TEST)
+
+
+TRACE_TEST(glxgears 238 6620)
```

Comment:
These traces are quite big.  I've been refraining from adding large traces as test cases.

First, double check you recompress with brotli (ie, `aptirace repack -b ...`) to reduce their size.  If you really need a git repos, then we should create a sub repos.  But it might be better to find different kind of storage (ie out of tree.)

Reply:
I've added a sub-repo and an option to enable the tests so that there is no need to always check out the tests.